requirements.txt

```diff
@@ -1,149 +1,138 @@
-anyio==3.7.1; python_full_version > "3.8.0" and python_version < "4.0" \
```

Comment:
**out of scope, question**: does anyone else also think that it would be nice if hashes are not included in requirements.txt file (lke we have done in some other projects?)